youdao-dictionary.el

```diff
@@ -84,6 +85,25 @@
 See URL `https://github.com/xuchunyang/chinese-word-at-point.el' for more info."
   :type 'boolean)
 
+(defcustom tip-function
+  (if (version< emacs-version "26.1")
+      #'-pos-tip
+    #'-posframe-tip)
```

Comment:
The posframe interface needs the following to work:

- Emacs 26.1
- posframe.el is installed
- Emacs runs in GUI mode

It's OK to not considering comprehensively here. I point it out in case you didn't notice.

Reply:
Thank you! Somehow I always had the impression that posframe.el also works for CLI. I checked and it is true that posframe.el will only work in GUI. Under this circumstance, the point you made in your previous comment is perhaps a better choice. I will update the code in a few hours.

Reply:
Updated.
- Remove the custom variable,
- Wrap posframe call in if,
- Updated documentation